doc/ADMIN.md

```diff
@@ -0,0 +1,4 @@
+* Email required for administation (uses "magic links" instead of admin passwords)
```

Comment:
I'm not sure to get what this is supposed to mean ... Does this mean you need to have a working email setup to access the webadmin ?

If so it sounds more like a disclaimer which should be displayed at the pre-installed step rather than in the admin doc

Reply:
Ah no, this mean it uses an email as login (it can be hosted anywhere, gmail if you want) and doesn't have passwords but magic links for login